A_Modelica/DroneLibrary/Mechanical/package.mo

```diff
@@ -38,5 +38,15 @@ package Mechanical
         extent={{4.0,-60.0},{14.0,-50.0}}),
       Line(
         origin={9.0,46.0},
-        points={{-10.0,-26.0},{72.0,-26.0},{72.0,-86.0},{-10.0,-86.0}})}));
+        points={{-10.0,-26.0},{72.0,-26.0},{72.0,-86.0},{-10.0,-86.0}})}),
+    Documentation(info="<html>
+<p>The models for the drone&apos;s motors and power sources are located in the <b>Mechanical</b> package. This package contains models for the following:</p>
+<ul>
+<li><a href=\"modelica://DroneLibrary.Mechanical.Blades\">Blades:</a> the mechanical models for the quadcopter&apos;s blades.</li>
+<li><a href=\"modelica://DroneLibrary.Mechanical.Propeller\">Propeller:</a> the top-level package of the propeller containing the models for the motor, rotor, and blades connected in different variations.</li>
+<li><a href=\"modelica://DroneLibrary.Mechanical.Chassis\">Chassis: </a>models of the quadcopter body.</li>
+<li><a href=\"modelica://DroneLibrary.Mechanical.Rotor\">Rotor:</a> the models of the rotor connecting the motor to the blades.</li>
+<li><a href=\"modelica://DroneLibrary.Mechanical.Motor\">Motor</a>: motor models to drive the blades to provied lift to the aircraft.</li>
+</ul>
```

Comment:
Instead of this list which is superfluous since Dymola will generate it automatically you might want to add description strings to the sub-packages instead such that the autogenerated list does not have a empty field there.  This is how it currently appears in Dymola:
![image](https://github.com/ALSETLab/Modelica-Drone-3D-FMI/assets/9332/16f466a4-0ea7-4b55-9b86-8be9559d7def)

Reply:
This has still not been fixed.

Reply:
Since we are running out of time I needed to do this :-( 2a35bda058876bd460f0808f3da1fa252b8e0672